.tool-versions

```diff
@@ -0,0 +1,4 @@
+golang 1.17.5
```

Comment:
Similar. I'm not sure we want to check in this file at the moment.

I do think this is a good topic though to raise in Slack. We need to have some better dev-tools & environment setup to make contributing easier. I'd be curious to get your thoughts on this topic if you want to start a thread.

Reply:
removed